src/runtime/particle-claim.ts

```diff
@@ -10,36 +10,44 @@
 
 import {HandleConnectionSpec} from './particle-spec';
 import {ParticleClaimIsTag, ParticleClaimDerivesFrom, ParticleClaimStatement} from './manifest-ast-nodes';
+import {browserLocalStorage} from '@tensorflow/tfjs-core/dist/io/local_storage';
```

Comment:
Huh? Why is this here? It isn't referenced in any of your edits below.

Reply:
Done